content/academy/pro_apify_scraper_development/integrating_webhooks.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Integrating webooks
+description: Learn how to integrate webhooks into your actors. Webhooks are a super powerful tool, and can be used to do almost anything!
+menuWeight: 6.4
+paths:
+    - pro-apify-scraper-development/integrating-webhooks
+---
+
+# [](#setup) Setup
```

Comment:
The main heading should be "Integrating webhooks", and then Setup should be a second-level heading, no?